pkg/kubelet/volumes.go

```diff
@@ -34,8 +34,6 @@ import (
 	"k8s.io/kubernetes/pkg/volume"
 )
 
-var errUnsupportedVolumeType = fmt.Errorf("unsupported volume type")
-
```

Comment:
It should not be needed, and there's no utility to keeping it around 'just in case'.  YAGNI